docs/user/02-10-usage.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Use BTP Manager to manage SAP BTP Service Operator 
+---
+
+You can install SAP BTP Service Operator: 
+<details>
+<summary>with a real BTP Manager Secret</summary>
+<br>
+
+To install SAP BTP Service Operator with a real BTP Manager Secret, follow these steps:
+1. Create ServiceBinding to obtain the access credentials to the ServiceInstance as described in points 2b and 2c of the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP Service Operator documentation.
+2. Copy the access credentials into the `hack/creds.json` file.
+3. Call [`create-secret-file.sh`](../../hack/create-secret-file.sh). 
+4. Apply the Secret in your cluster. 
+ 
+   ```sh
+   ./hack/create-secret-file.sh
+   kubectl apply -f deployments/prerequisites.yaml
+   kubectl apply -f hack/operator-secret.yaml
+   kubectl apply -f examples/btp-operator.yaml
+   ```
+   </details>
+  
+<details>
+<summary>with a dummy BTP Manager Secret</summary>
+<br>
+
+To install SAP BTP Service Operator with a dummy BTP Manager Secret, run the following commands:
+```sh
+kubectl apply -f deployments/prerequisites.yaml
+kubectl apply -f examples/btp-manager-secret.yaml
+kubectl apply -f examples/btp-operator.yaml
+```
+</details>
+<br>
+
+To check the `BtpOperator` CR status, run the following command:
+```sh
+kubectl get btpoperators btpoperator
+```
+
+The expected result is:
+```
+NAME                 STATE
+btpoperator          Ready
+```
+
```

Comment:
please add here a section and mention that from this point of time user can create serviceinstances and servicebindings. See: https://kyma-project.io/docs/kyma/latest/03-tutorials/00-service-management/smgt-01-deploy-service/ 
points 4,5,6,7,8, 9(first two commands do cleanup instance and binding)